pkg/kube/teams.go

```diff
@@ -10,11 +11,39 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
-// GetAdminNamespace tries to find the namespace which is annotated as the global admin namespace for the cluster
-// or returns the current namespace
-func GetAdminNamespace(kubeClient kubernetes.Interface, ns string) (string, error) {
-	// TODO find the admin namespace via a label on the current dev namespace - or use current?
-	return ns, nil
+const adminNamespaceAnnotation = "jenkins-x.io/admin-namespace"
+
+// GetAdminNamespace tries to find the admin namespace that corresponds to this team.
+// in other words this is the namespace where the team CRD was initially created when this team was created,
+// or the current team namespace for the case where this team was just created with a standalone `jx install`
+func GetAdminNamespace(kubeClient kubernetes.Interface, teamNs string) (string, error) {
+	namespace, err := kubeClient.CoreV1().Namespaces().Get(teamNs, metav1.GetOptions{})
+	if err != nil {
+		return "", errors.Wrapf(err, "obtaining the team namespace (%s) when getting the admin namespace", teamNs)
+	}
+	adminNs := namespace.Annotations[adminNamespaceAnnotation]
```

Comment:
nit: 
```Go
adminNs, ok := ...
if ok && adminNs != "" {
...
}
```

Reply:
given I need to treat an empty string the same as not present for this case I don;t think it is needed.